server/gamenode/Lobby.ts

```diff
@@ -26,6 +25,7 @@ export class Lobby {
     private users: LobbyUser[] = [];
     private tokens: { battleDroid: any; cloneTrooper: any; experience: any; shield: any };
     private gameChat: GameChat;
+    private lobbyOwner: string;
```

Comment:
Since we have multiple ways of identifying a user, probably good to be explicit and call this `lobbyOwnerId`